logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LogoutUseCase.kt

```diff
@@ -16,13 +18,19 @@ class LogoutUseCase(
     private val userId: QualifiedID,
     private val authenticatedDataSourceSet: AuthenticatedDataSourceSet,
     private val clientRepository: ClientRepository,
-    private val mlsClientProvider: MLSClientProvider
+    private val mlsClientProvider: MLSClientProvider,
+    private val authenticatedDataSourceSetProvider: AuthenticatedDataSourceSetProvider = AuthenticatedDataSourceSetProviderImpl
```

Comment:
I understand it's odd to have `AuthenticatedDataSourceSet` and `AuthenticatedDataSourceSetProvider` injected here, but as I mentioned, this is subject to change and I wanted to avoid spending time on it.